API/Controllers/OrdersController.cs

```diff
@@ -138,5 +147,16 @@ public async Task<ActionResult> CompleteOrder(int id)
 
             return NoContent();
         }
+
+
+        private float calculateTotalPrice(int area, IEnumerable<ServicePrice> services)
```

Comment:
Jak rozumiem to jest po to, aby gdy jest zamówienie cykliczne klient wiedział ile zapłaci za całość ?

Reply:
Nie, w momencie tworzenia pojedynczego zamówienia jest wyliczany koszt na podstawie metrażu i usług, a potem jest przypisywany do TotalPrice.

Reply:
Zmieniłem to w ten sposób, ponieważ wyliczanie w locie za każdym razem po pierwsze jest mało optymalne, a po drugie jakby się zmieniły ceny usług to zmieniłby się koszt zamówień z przeszłości co jest nielogiczne.

Reply:
Ten drugi argument mnie przekonuje.